main.py

```diff
@@ -1,17 +1,16 @@
 import os, json, werkzeug
-import logging 
 from flask import Flask, request
 from werkzeug.security import generate_password_hash, check_password_hash
+from credit import load_credits
 
 
-logging.basicConfig(filename="main.log", 
-                    format='%(asctime)s %(message)s', 
-                    filemode='w') 
-logger = logging.getLogger() 
-logger.setLevel(logging.ERROR) 
+credit = load_credits(os.path.dirname(os.path.realpath(__file__)) + '/credentials.txt')
 
 def login():
-  
+    
+    if type(credit) is not dict:
+        return 'Server currently are unavailable.', 500
```

Comment:
@nkartashov Bro, take a look. Here I load credentials to credit variable. In case credit.py has some problems with credentials.txt in def login() I check that type of credit is dict. But in test I can't figure out how to mock type of variable. Maybe mock just can't do it. How do you think?